diff --git a/node_modules/@holochain/tryorama/lib/conductor.js b/node_modules/@holochain/tryorama/lib/conductor.js
index bddda32..3faacb3 100644
--- a/node_modules/@holochain/tryorama/lib/conductor.js
+++ b/node_modules/@holochain/tryorama/lib/conductor.js
@@ -79,8 +79,7 @@ export class Conductor {
         if (options?.bootstrapServerUrl) {
             args.push("--bootstrap", options.bootstrapServerUrl.href);
         }
-        args.push("webrtc");
-        args.push(signalingServerUrl.href);
+        args.push("mem");
         defaultLogger.debug("spawning hc sandbox with args:", args);
         const createConductorProcess = spawn("hc", args);
         createConductorProcess.stdin.write(LAIR_PASSWORD);
